layouts/community/ergodox/azurefog/keymap.c

```diff
@@ -0,0 +1,385 @@
+#include "ergodox_ez.h"
+#include "debug.h"
+#include "action_layer.h"
+#include "version.h"
+
+
+enum custom_keycodes {
+  PLACEHOLDER = SAFE_RANGE, // can always be here
```

Comment:
You don't actually need a placeholder here.  It's absolutely fine being:
```suggestion
  RGB_SLD= SAFE_RANGE, // can always be here
```